api/services_cluster.go

```diff
@@ -59,6 +60,35 @@ func removeClusterMember(state state.State, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
+	ceph := sh.Services[types.MicroCeph]
+	if ceph != nil {
+		// If we got a 503 error back, that means the service is installed, but hasn't been set up yet, so there are no cluster members to remove.
```

Comment:
I understand this situation can happen if you form a MicroCloud without MicroCeph snap installed, and then at a later point in time install MicroCeph on this member and try to remove it afterwards?